src/status_im2/contexts/communities/home/style.cljs

```diff
@@ -42,3 +43,34 @@
    :left        0
    :padding-top top})
 
+;;;; CARD ANIMATION
+(def card-bottom-override {:margin-bottom 16}) ; Original 8 + 8 from tabs top padding
+(def card-height (+ 56 16)) ; Card height + its vertical margins
+(def card-total-height (+ card-height 8)) ; added 8 from tabs top padding
+(def card-opacity-factor (/ 100 card-height 100))
+
+(defn- value-in-range
+  "Returns `num` if is in the range [`lower-bound` `upper-bound`]
+  if `num` exceeds a given bound, then returns the bound exceeded."
+  [num [lower-bound upper-bound]]
```

Comment:
If possible, I would avoid the name `num` because it's a core Clojure JVM function https://clojure.github.io/clojure/clojure.core-api.html#clojure.core/num. Maybe in the future CLJS will also have this function?